### DISCUSSION TITLE
Show us the content of one's sack of sacks

### DISCUSSION THREAD

**Discussion by skyblockz** [HypixelDev/PublicAPI#256] [Ideas]
Since the latest sacks update is there, it would be nice to include the following things:

1. The content of Sack of Sacks 
2. The content of X sack (the amount of contained items)
3. (Personal Compactor) The configured item for PCs

**Comment by Flachdachs** [comment]
The content of a specific sack is available in `sacks_counts` of a player's profile. However, the content of the Sack of Sacks is currently not available. I'm interested in this data too, or at least the maximum capacity. The sacks in the inventory also count to this capacity.
